roles/bridge/tasks/setup_zfs.yml

```diff
@@ -32,18 +32,22 @@
   changed_when: false
 
 - name: Retrieve loopback device
-  ansible.builtin.shell:
-    cmd: |
-      set -o pipefail
-      losetup -a | grep {{ bridge_zfs_image_path }} | cut -d: -f1
+  ansible.builtin.shell: |
+    set -o pipefail
+    losetup -a | grep {{ bridge_zfs_image_path }} | cut -d: -f1
   register: loop_device
   changed_when: false
+  args:
+    executable: /bin/bash
 
 - name: Create ZFS pool with mount point
   ansible.builtin.command:
     cmd: zpool create -m {{ bridge_zfs_pool_mount_point }} {{ bridge_zfs_pool_name }} {{ loop_device.stdout }}
-    creates: "{{ bridge_zfs_pool_mount_point }}"
-  when: loop_device.stdout is defined
+  changed_when: false
```

Comment:
I think creates is better, it sort of defines what do we expect this command to change or do

Reply:
Any reason to remove it ?

Reply:
It's creating the creates folder and skipping the creation of zpool